falcon/util/mediatypes.py

```diff
@@ -213,13 +213,11 @@ def match_score(self, media_type: _MediaType) -> Tuple[int, int, int, int, float
 
         return (main_matches, sub_matches, exact_match, len(matching), self.quality)
 
-    def __repr__(self) -> str:
-        q = self.quality
-        return f'MediaRange<{self.main_type}/{self.subtype}; {q=}; {self.params}>'
-
 
 # PERF(vytas): It is possible to cache a classmethod too, but the invocation is
 #   less efficient, especially in the case of a cache hit.
+# NOTE(vytas): Also, if we decide to make these classes public, we either need
+#   to keep these cached parsers private, or to make sure we use frozen classes.
```

Comment:
not 100% accurate, since params is mutable. likely ok to just note
> Also, if we decide to make these classes public, we need to keep these cached parsers private.

Reply:
Good point! I'll address it in the tweaks PR.